container/push.bzl

```diff
@@ -187,6 +187,11 @@ container_push = rule(
             default = Label("//container:push-tag.sh.tpl"),
             allow_single_file = True,
         ),
+        "robot_email": attr.string(
+            default = "",
```

Comment:
Move to line 159

Reply:
Created a new commit to move it, both here and contrib/push-all.bzl.